src/Nethermind/Nethermind.Consensus/Producers/PayloadAttributes.cs

```diff
@@ -25,6 +26,7 @@ public class PayloadAttributes
 
     public IList<Withdrawal>? Withdrawals { get; set; }
 
+    public Keccak? BeaconStateRoot { get; set; }
```

Comment:
Similar to what is discussed here: https://github.com/NethermindEth/nethermind/pull/5845
I think we should have `PayloadAttributesV3` now with the added fields?